### PR TITLE
More documentation fixes

### PR DESCRIPTION
**Pull request by tdeboissiere**

### CODE REVIEW

**Comment by MaxBenChrist** [comment]
Thanks for the pr!

File: docs/text/feature_filtering.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ To limit the number of irrelevant features, tsfresh deploys the fresh algorithm
 based on Scalable Hypothesis tests`) [1]_.
 
 The algorithm is called by :func:`tsfresh.feature_selection.feature_selector.check_fs_sig_bh`.
-It is a efficient, scalable feature extraction algorithm, which filters the available features in an early stage of the
+It is a nefficient, scalable feature extraction algorithm, which filters the available features in an early stage of the
```
Comment (MaxBenChrist) [inline docs/text/feature_filtering.rst]: nefficient?

File: tsfresh/examples/driftbif_simulation.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ class velocity(object):
     1
     >>> print(ds.deterministic) # Equilibrium velocity
     0.0015191090506254991
-    >>> v = ds.simulate(20000) # Simulate velocity time series with 20000 time steps being disturbed by Gaussian white noise
+    >>> v = ds.simulate(20000) # Simulate velocity. Time series with 20000 time steps being disturbed by Gaussian white noise
```
Comment (MaxBenChrist) [inline tsfresh/examples/driftbif_simulation.py]: it is a time series of velocity, maybe write "Simulated velocity as a time series with ..."

File: tsfresh/feature_extraction/feature_calculators.py [hunk 3]
```diff
@@ -94,8 +94,7 @@ def decorate_func(func):
 @not_apply_to_raw_numbers
 def variance_larger_than_standard_deviation(x):
     """
-    Boolean variable denoting if the variance of x is greater than its standard deviation. Is equal to variance of x
-    being larger than 1.
+    Boolean variable denoting if the variance of x is greater than its standard deviation.
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: I would rather not delete the second sentence.

File: tsfresh/feature_extraction/feature_calculators.py [hunk 4]
```diff
@@ -109,13 +108,13 @@ def variance_larger_than_standard_deviation(x):
...
     """
-    Boolean variable denoting if the variance of x is higher than half of the range, calculated as the half the
-    difference between max and min of x.
+    Boolean variable denoting if the standard dev of x is higher
+    than 'r' times the range = difference between max and min of x.
     Hence it checks if
 
     .. math::
 
-        | std(x) | > r * (max(X)-min(X))
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: :D

File: tsfresh/feature_extraction/feature_calculators.py [hunk 5]
```diff
@@ -683,9 +687,9 @@ def ratio_value_number_to_time_series_length(x):
 @not_apply_to_raw_numbers
 def fft_coefficient(x, c, param):
     """
-    Calculates the fourier coefficients of the one-dimensional discrete Fourier Transform for real input by fast
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: why did you split this up into three sentences?

Comment (tdeboissiere) [inline tsfresh/feature_extraction/feature_calculators.py]: Just for clarity. (Smaller lines are usually more readable). Changed it back to original

**Comment by tdeboissiere** [comment]
All requested changes have been implemented

**Comment by MaxBenChrist** [comment]
Awesome, it must have taken you quite some time to find all those errors and typos. Thank you @tdeboissiere :)

**Comment by MaxBenChrist** [comment]
Are you aware that I edited my last comment, @tdeboissiere ? I will merge this pr as soon as you delete the output of that notebook.

**Comment by tdeboissiere** [comment]
Cleared the notebook and corrected the spelling of activity.

**Comment by MaxBenChrist** [comment]
👍

**Pull request merged**